MetaMask/metamask-extension
# UX: Send Flow: Add 'Sent to Contract' warning to new send page

## **Description**

This PR implements the "You're sending to a contract" acknowledgement in the new send flow.  This existed in the old send flow.

## **Related issues**

N/A

## **Manual testing steps**

1. Go to the new send page
2. As the recipient, paste: `0x514910771af9ca656af840dff83e8264ecf986ca` ; this is the $LINK token address
3. See the warning, note that you cannot submit the send until acknowledgement

## **Screenshots/Recordings**



### **Before**

N/A

### **After**

<img width="415" alt="SCR-20240116-mthm" src="https://github.com/MetaMask/metamask-extension/assets/46655/87955c6d-262b-4c60-82c6-71a295f42e41">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
